pages/events/fall-fest.vue

```diff
@@ -0,0 +1,230 @@
+          hid: 'twitter:description',
+          name: 'twitter:description',
+          content: description
+        },
+        {
+          hid: 'twitter:image',
+          name: 'twitter:image',
+          content: image
+        },
+        {
+          hid: 'twitter:image:alt',
+          name: 'twitter:image:alt',
+          content: title
+        },
+        {
+          hid: 'og:title',
+          property: 'og:title',
+          content: title
+        },
+        {
+          hid: 'og:description',
+          property: 'og:description',
+          content: description
+        },
+        {
+          hid: 'og:image',
+          property: 'og:image',
+          content: image
+        },
+        {
+          hid: 'og:image:secure_url',
+          property: 'og:image:secure_url',
+          content: image
+        },
+        {
+          hid: 'og:image:alt',
+          property: 'og:image:alt',
+          content: title
+        }
+      ]
+    }
+  },
+  components: { FallFestFaqSection, UniversityDirectorySection },
+  layout: 'default-max'
+})
+export default class SummerSchoolPage extends QiskitPage {
+  routeName = 'fall-fest'
+  agendaColumnsDataTable: string[] = ['Day', 'University', 'Type', 'Link']
+  headerData = header
+  agendaData = agenda
+  helpfulResourcesData = helpfulResources
+}
+</script>
+
+<style lang="scss" scoped>
+.fall-fest-page {
+  display: flex;
+  flex-direction: column;
+
+  &__section {
+    margin-top: $spacing-10;
+    margin-bottom: $spacing-07;
+
+    @include mq($until: large) {
+      margin-bottom: $spacing-05;
+    }
+  }
+
+  &__content {
+    @include mq($until: large) {
+      max-width: 100%;
+    }
+  }
+}
+</style>
+
+<style lang="scss" scoped>
+// overrides
+.fall-fest-page {
+  ::v-deep {
+    // TODO: Extract styles like "_checkbox" to be defined globally
+    & a.bx--tabs__nav-link {
+      color: $text-color-light;
+      border-bottom-color: $border-color;
+
+      &:focus,
+      &:active {
+        outline: none;
+      }
+
+      &:not(.bx--tabs__nav-item--disabled) {
+        color: $text-color-light;
+      }
+    }
+
+    // TODO: Not sure if the order is important in these selectors.
+    // So, disabling. Review the reule 'no-descending-specificity' here:
+    // https://stylelint.io/user-guide/ignore-code
+    // stylelint-disable no-descending-specificity
+    & .bx--tabs__nav-item {
+      &:hover:not(.bx--tabs__nav-item--selected):not(.bx--tabs__nav-item--disabled) .bx--tabs__nav-link,
+      &:not(.bx--tabs__nav-item--selected):not(.bx--tabs__nav-item--disabled):not(.bx--tabs__nav-item--selected) .bx--tabs__nav-link:focus,
+      &:not(.bx--tabs__nav-item--selected):not(.bx--tabs__nav-item--disabled):not(.bx--tabs__nav-item--selected) a.bx--tabs__nav-link:active,
+      &:not(.bx--tabs__nav-item--disabled) .bx--tabs__nav-link, .bx--tabs__nav-item--selected:not(.bx--tabs__nav-item--disabled) .bx--tabs__nav-link:focus {
+        color: $text-color-light;
+      }
```

Comment:
not sure if this is needed? I tried simplifying the declaration, but didn't notice any difference when this whole block is removed...?

Reply:
yeah this isn't attractive target selecting, but historically, it was used to override the default styling...


https://user-images.githubusercontent.com/6276074/190221733-9ee4ace6-596c-4940-aa5d-2cdae3a90896.mov

Reply:
meant to reply in this thread 🙇 https://github.com/Qiskit/qiskit.org/pull/2783#discussion_r971110566
^also not blocking, just potentially less code 😅

Reply:
your suggestion worked, thank you 🙇🏼‍♂️